source/technical/reservations.rst

```diff
@@ -128,6 +133,44 @@ You may reserve a specific node by providing its *UUID*. To learn more about how
 
 .. _reservations-extend-lease-gui:
 
+Creating a Lease to Reserve a VLAN Segment
+__________________________________________
+
+On the *Create Lease* window, start filling the form as if reserving physical
+nodes: pick a name, start time, and lease duration. Then:
+
+#. Change the *Resource Type* field to *Network*.  The control to configure the
+   number of hosts will disappear and be replaced by a field to enter the
+   network name.
+
+#. Enter a name for the Neutron network which will be created by Blazar on the
```

Comment:
As I understand, Blazar will create the Neutron automatically once the lease is created. I am wondering about what options and parameters are used for network creation. For named VFCs (Virtual Switch Foo) case, neutron plugin utilizes `--description` field to specify user owned OpenFlow controllers and the name of the VFCs. Does the lease have fields for these options?

Reply:
Yes, see https://github.com/ChameleonCloud/docs/pull/24/files#diff-13121cc13097a95fecd38b527e0c98afR381